### DISCUSSION TITLE
关于ImageOrigin 的问题

### DISCUSSION THREAD

**Discussion by dugufly** [Tencent/libpag#1752] [Q&A] [answered]
请问一下，通过replaceImageByName用纹理替换PAG文件中图片，我发现ImageOrigin 中只有两个方向，如果输入纹理方向是任意的（带矩阵），是不是需要先转换一个方向才能使用这个函数？

`            mPagFile->replaceImageByName(strLayerName,  pag::PAGImage::FromTexture(texture, pag::ImageOrigin::BottomLeft));
`

**Comment by kevingpqi123** [comment, marked as answer]
关于占位图填充，可以看下这篇文章：
https://pag.art/docs/pag-fillmode.html

**Comment by dugufly** [comment]
如果通过PAGImage.setMatrix（） 设置外部矩阵，矩阵的原点是在图像的左下角吧？
